City-of-Helsinki/drupal-hdbt
# UHF-X Fix content liftup image placeholder position on large cards

How to test:

1. Checkout the branch on hdbt.
2. Clear caches.
3. Go to a landing page for example and add a content liftup with some unit that has plenty of information but no image and remember that the unit needs to be published before you can add it as content liftup.
4. View the added content liftup with narrow browser window so that you can see that the placeholder image is always on center and the background color is added above and below so that white color is never shown.
